test_regress/t/t_enum_huge_methods.pl

```diff
@@ -11,6 +11,7 @@
 scenarios(simulator => 1);
 
 compile(
+    verilator_flags2 => ["-Wno-BLKANDNBLK"]
```

Comment:
Please instead put lint_off/lint_on brackets around the relevant lines in the verilog, thanks.  Then feel free to squash merge after tests pass.

Reply:
Done, thanks.